src/app.module.ts

```diff
@@ -23,8 +27,14 @@ if (!fs.existsSync(envFilePath)) {
   imports: [ConfigModule.forRoot({
     isGlobal: true,
     envFilePath,
-  })],
+  }),
+  MongooseModule.forRoot('mongodb://115.146.86.210:27017/testdb'),
```

Comment:
I have add it to environment variables, please use `process.env.DATABASE_CONN_STRING` instead of plain text. 
In your local machine, I would recommend you store export it as a environment variables or add it to `.env/.development.env` and never push it. because current repo is a public repo.